hs_labels/migrations/0003_manual_delete_duplicates.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+def delete_duplicates(apps, schema_editor): 
+    # delete oldest copies of records from UserResourceLabels
+    UserResourceLabels = apps.get_model("hs_labels", "UserResourceLabels")
+    oldrecord = None 
+    # order by reverse start so that the first object should be preserved
+    for record in list(UserResourceLabels.objects.all()\
```

Comment:
Yes, because we are deleting rows. The default is to iterate abstractly over the QuerySet. We wish to avoid that because we are deleting things and that has some small chance of screwing up iteration.